app/kafka/consumer.py

```
@@ -0,0 +1,113 @@
+            if event.event_type == order_events_pb2.ORDER_CREATED:
+                process_order_created(event.order_created)
+        except DecodeError as e:
+            print("Error al decodificar mensaje:", e)
+        except Exception as e:
+            print("Error inesperado:", e)
+
+
+def process_order_created(order_created):
+    print(f"\nProcesando orden: {order_created.order_id}")
+
+    payment_success = random.choice([True, False])
```

Ruff (0.13.3)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)